core/src/main/scala/cr/pulsar/internal/FutureLift.scala

```diff
@@ -16,45 +16,11 @@
 
 package cr.pulsar.internal
 
-import cats.effect._
-import cats.syntax.all._
-import cats.effect.implicits._
-import java.util.concurrent._
+import monix.catnap
+import java.util.concurrent.{ CompletableFuture, _ }
```

Comment:
Auto-importing mess up?. The `_` is no longer needed, just leave the `CompletableFuture`.